lib/pwwka/configuration.rb

```diff
@@ -137,5 +143,14 @@ def omit_payload_from_log?(level_of_message_with_payload)
       return true if @receive_raw_payload
       Pwwka::Logging::LEVELS[Pwwka.configuration.payload_logging.to_sym] > Pwwka::Logging::LEVELS[level_of_message_with_payload.to_sym]
     end
+
+    private
+
+    def background_jobs
```

Comment:
I tried to make this a constant at the top of this file, but `Pwwka::SendMessageAsyncJob` has a dependence on `Pwwka::Configuration` so there was a weird circular dependency thing going on.